lib/cog.ex

```diff
@@ -29,20 +29,20 @@ defmodule Cog do
   def site_namespace, do: "site"
 
   defp build_children(:dev, nochat, _) when nochat != nil do
```

Comment:
We talked about it but I don't think we ever circled back on it. I can nuke all that in a second PR.